packages/admin/cms-admin/src/contentScope/ContentScopeIndicator.tsx

```diff
@@ -25,7 +25,7 @@ const ScopeIndicator = styled("div", { shouldForwardProp: (prop) => prop !== "gl
     position: ${({ variant }) => (variant === "toolbar" ? "fixed" : "absolute")};
     top: -12px;
     left: 0;
-    z-index: 1201;
+    z-index: 1200;
```

Comment:
I think the z-index should be dependent von the MUI z-indices. Could be `theme.zIndex.drawer - 1` or `theme.zIndex.appBar` (if MasterLayout also uses appBar)

Reply:
I've updated the zIndex, so now it is dependent of the drawer.